pkg/yurttunnel/server/serveraddr/addr_test.go

```diff
@@ -0,0 +1,29 @@
+package serveraddr
+
+import (
+	"testing"
+	"github.com/stretchr/testify/assert"
+	"fmt"
+)
+
+func TestGetDefaultDomainsForSvc(t *testing.T) {
+	domains := getDefaultDomainsForSvc("", "")
```

Comment:
you can add a new test case or a sub test case for ns and name empty setting.

Reply:
I modified it